src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Square extends Figures {
+    private int side;
+    private final String name = "square";
+
+    public void setSide(int side) {
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void draw(Figures figure) {
+        System.out.println(figure);
```

Comment:
```suggestion
        System.out.println(this);
```
в такому випадку ти можеш просто винести цей метод в Figure  і то буде твоя реалізація за замовчуванням.. Якщо захочеш її перевизначити, зможеш то зробити в наслідниках